central_v2/src/hooks/useCentralDataManager.tsx

```diff
@@ -62,7 +68,18 @@ export default function useCentralDataManager({
   gameQuestion
 }: UseCentralDataManagerProps): UseCentralDataManagerReturnProps {
   const apiClients = useTSAPIClientsContext(APIClientsContext);
+  const userProfile = useUserProfileContext(UserProfileContext);
```

Comment:
`userProfileContext` etc will get removed in the next PR, so ignore this for now